src/pp_ast.mli

```diff
@@ -60,28 +60,46 @@ module Config : sig
           be. *)
 end
 
+type simple_val =
```

Comment:
You're not responsible for this as I'm the one who came up with that name originally but since we're going to expose it, I think we should come up with something better.

I think some commonly used name for this kind of type is `repr` as in representation. If you have a suggestion I'd gladly hear it as I'm not super inspired here as the original name suggests :smile:

Reply:
`repr` looks like a good name, though.